src/components/PlaceholderContainer/PlaceholderContainer.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+
+import {Button} from '../Button';
+import {block} from '../utils/cn';
+
+import {componentClassName} from './constants';
+import type {PlaceholderContainerActionProps, PlaceholderContainerProps} from './types';
+
+import './PlaceholderContainer.scss';
+
+const b = block(componentClassName);
+
+const PlaceholderContainerAction = (props: PlaceholderContainerActionProps) => {
+    return (
+        <div className={b('action')}>
+            <Button
+                className={b('action-btn')}
+                view={props.view || 'normal'}
+                size={props.size || 'm'}
+                loading={Boolean(props.loading)}
+                disabled={Boolean(props.disabled)}
+                href={props.href || ''}
+                {...(props.onClick ? {onClick: props.onClick} : {})}
+            >
+                {props.text}
+            </Button>
+        </div>
+    );
+};
+
+export const PlaceholderContainer = ({
+    direction = 'row',
+    align = 'center',
+    size = 'l',
+    className = b(),
```

Comment:
This className should be an addition to `b()`, not replacement: `b(mods, className)`